dev/NavigationView/NavigationView_InteractionTests/NavigationViewTests.cs

```diff
@@ -4554,6 +4554,30 @@ public void VerifyCorrectVisualStateWhenClosingPaneInLeftDisplayMode()
             }
         }
 
+        [TestMethod]
+        [TestProperty("TestSuite", "E")]
+        public void VerifyHoldingKeyOnlyInvokesOnce()
+        {
+            using (var setup = new TestSetupHelper(new[] { "NavigationView Tests", "NavigationView compact pane length test" }))
+            {
+                Verify.AreEqual("0", GetInvokeCount());
+                var homeItem = FindElement.ByName("NavViewInvokeItem");
+                FocusHelper.SetFocus(homeItem);
+
+                KeyboardHelper.PressKey(Key.Enter);
+
+                Verify.AreEqual("1", GetInvokeCount());
+
+                // Figure out to hold a key ...
```

Comment:
>// Figure out to hold a key ... [](start = 16, length = 31)

I think you will need to end up calling SendKeys(['Enter Down'])  to do this you'll probably need to write a new helper in KeyboardHelper

Reply:
Updated the KeyBoardHelper to also have a helper for holding down a key and updated test to test what we need.